src/redux/reducers/index.ts

```diff
@@ -1,5 +1,9 @@
 //**Make new file for your reducer then */
 //**combine all the reducers in here */
-const createRootReducer = () => combineReducers();
+import {combineReducers} from 'redux'
+import {counterReducer} from './counter'
+const createRootReducer = () => combineReducers({
+    counterReducer
```

Comment:
You could name it as **counter** for short:
`counter: counterReducer`

Reply:
okay, i will, Thanks